### PR TITLE
Fix build for Rust 1.0

### PR DESCRIPTION
**Pull request by bombless**
I think what they do in nalgebra is kinda weird because we already have associated types, but anyway this fix the build for stable Rust.

### CODE REVIEW

**Comment by sebcrozet** [comment]
Thanks, but, what version of rustc are you using exactly ? I'm currently running rust stable 1.0 (`rustc 1.0.0 (a59de37e9 2015-05-13) (built 2015-05-14`) and `ncollide` compiles without this patch.

`nalgebra` does some weird things with associated types (and sometimes does not use them) because they are not yet completely supported by the compiler: I get ICEs when I try to use them.

**Comment by bombless** [comment]
I don't know, but maybe you should try `cargo clean && cargo build`, see https://github.com/sebcrozet/ncollide/issues/78

I don't have strong opinion on this, I just saw people talking about not able to build it on IRC and come to see what I can do.

I use rustc compiled from master branch pulled one day ago by the way.

**Comment by bombless** [comment]
I see, maybe the break come from nalgebra.
Then we can desinate the version of nalgebra in `Cargo.toml`, if it do sane vesioning.

**Comment by sebcrozet** [comment]
I don't recommend people to use the `master` branch of the compiler or even the nightlies. I believe the common approach now is to stay with the latest stable releases.

**Comment by bombless** [comment]
btw, if you are using low version of nalgebra, you may need `cargo update` to reproduce the problem, see https://crates.io/crates/nalgebra

**Comment by bombless** [comment]
tl;dr: check if you are using nalgebra 0.2.17

**Comment by bombless** [comment]
@sebcrozet I'm very sorry that I didn't check it carefully.
Already filed an issue for it, https://github.com/rust-lang/rust/issues/26279
